drivers/performance_measurement/app/models/performance_measurement/details.rb

```diff
@@ -269,7 +271,25 @@ def show_row?(key)
         oph_average_bed_utilization: :oph,
       }
     end
-
+    # The detail_hash method defines the complete set of performance metrics used in the dashboard.
+    # Each key represents a specific performance measure (e.g., count_of_homeless_clients) with
+    # attributes that control how it's calculated, displayed, and compared to goals.
+    #
+    # Key attributes:
+    # - category/sub_category: Classification ('Rare', 'Brief', 'Non-Recurring') and subgroup
+    # - column: Where metric is displayed (:system, :project, or :both)
+    # - year_over_year_change: Whether to display year-over-year comparison
+    # - title: Human-readable title
+    # - goal_description: Description with %{goal} placeholder
+    # - goal_direction: Direction of improvement ('>', '<', '+', or '-')
+    # - goal_calculation: Method reference for goal calculation
+    # - calculation_description: How the metric is calculated
+    # - calculation_column: Client data column(s) for calculation
+    # - detail_columns: Client-level columns for drill-down views
```

Comment:
seems like these are for drill downs as opposed to the client_project

Reply:
I believe drill-downs are only shown for client_project (project-level) data.  So, yes, but in relation to `client_project`)